server/src/main/java/io/deephaven/server/session/TicketResolver.java

```diff
@@ -148,11 +149,21 @@ default <T> void publish(
                 .submit(source::get);
     }
 
+    /**
+     * Retrieve the ByteString Schema for a given FlightDescriptor.
+     *
+     * @param descriptor the flight descriptor to retrieve a schema for
+     * @param logId an end-user friendly identification of the descriptor should an error occur
+     * @return a FlightInfo describing this flight
+     */
+    SessionState.ExportObject<ByteString> getSchema(@Nullable SessionState session, Flight.FlightDescriptor descriptor,
```

Comment:
Can we have this return the `pojo.Schema` and have the router offer both `getSchema` and `getSchemaBytes`?

Reply:
Good suggestion, more generally that is in line with https://github.com/deephaven/deephaven-core/issues/6295